.ci/targets/web_platform_tests.yaml

```diff
@@ -0,0 +1,9 @@
+tasks:
+  - name: prepare tool
+    script: .ci/scripts/prepare_tool.sh
```

Comment:
is `prepare_tool.sh` responsible for installing Chromedriver and putting it in the `$PATH`?

Reply:
Nope, that's all done automagically by [this line](https://github.com/flutter/packages/blob/d57cd748b8531b2538f24cb18f22e7ce1cd94aa7/.ci.yaml#L47) in the LUCI world.

Reply:
Oh yeah, you're right! I had forgotten all about that (and I've probably fiddled with the script that generates those in the past for flutter/engine 🤦)